openfisca_france_local/communes/antony/antony_aide_depart_sejour_adapte.py

```diff
@@ -13,9 +13,9 @@ def formula(famille, period, parameters):
 
         condition_ressources_remplies = famille('antony_eligibilite_ressources', period)
 
-        handicap = famille.members('handicap', period)
+        nb_handicap = famille.sum(famille.members('handicap', period), role = Famille.ENFANT)
```

Comment:
Du coup c'est les enfants handicapés ici non ?
```suggestion
        nb_enfants_handicapes = famille.sum(famille.members('handicap', period), role = Famille.ENFANT)
```